cmd/manager/main.go

```diff
@@ -19,78 +19,68 @@ package main
 import (
```

Comment:
Should this file be moved in the root directory?

Reply:
no, I don't think so. Since we are shipping multiple binaries with this project I think it makes more sense organizationally to keep them all under `cmd/<binary name>/`

Reply:
Wouldn't kubebuilder complain when generating new controllers or types if it can't find a main.go in the root?

Reply:
Yep it does, but I'm not terribly concerned by it. Generating new apis put them in `api/` instead of `/pkg/apis/`.

Moving main.go is going to require changing paths and some Bazel. I can see this task falling into the scope of this PR. I'd prefer doing it after due to the build changes, but if you'd like it done in this PR that's fine too.

Reply:
That's fine, I was mostly wondering if it was on your radar, rather than pushing it for doing it in the same PR